task10.rb

```diff
@@ -0,0 +1,24 @@
+# Return the maximum product of three numbers in an array, so [-8, -9, 1, 2, 7] outputs 504 (which is -8*-9*7)
+
+def task10(arr)
+  raise ArgumentError, "Not a valid format. Need an array" if arr.class != Array
+  raise ArgumentError, "Array is empty. This is not valid" if arr.empty?
+  raise ArgumentError, "Not a valid format. Need an array with more then two numbers inside" if arr.length < 3 
+  arr.each do |number|
```

Comment:
@MaraEfimova Not a mistake but you're not sure if this is a `number`, so until you are sure - it is still just an `element` :wink: